apps/src/gamelab/GameLabSprite.js

```diff
@@ -235,7 +235,7 @@ var AABBops = function (p5Inst, type, target, callback) {
         print("busted");
         return false;
       }*/
-      if(this.collider != undefined && other.collider != undefined)
+      if(this.collider !== undefined && other.collider !== undefined)
```

Comment:
Should we just be eslint ignoring this chunk of code since it's apparently copied from elsewhere?